.circleci/config.yml

```diff
@@ -113,26 +113,26 @@ jobs:
           command: rustc --version; cargo --version; rustup --version; rustup target list --installed
       - restore_cache:
           keys:
-            - cargocache-v2-package_vm_singlepass-rust:nightly-{{ checksum "Cargo.lock" }}
+            - cargocache-v2-package_vm_cranelift-rust:1.41.1-{{ checksum "Cargo.lock" }}
       - run:
           name: Build
           working_directory: ~/project/packages/vm
-          command: cargo build --locked
+          command: cargo build --locked --no-default-features --features "default-cranelift iterator"
```

Comment:
Please build/test without iterator.

I have had failing tests before when I didn't place the `#[cfg(feature="iterator")]` correctly. And they would pass with iterator, but compile errors without.

The non-iterator build/tests only need to run in one engine (not both) as it is orthogonal to the backend choice.

Reply:
Okay, looking at the test runtime, I add this to singlepass